src/test/java/com/salesforce/mirus/metrics/MirrorJmxReporterTest.java

```diff
@@ -0,0 +1,49 @@
+package com.salesforce.mirus.metrics;
+
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.TimeUnit;
+import org.apache.kafka.common.TopicPartition;
+import org.apache.kafka.common.metrics.Metrics;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+public class MirrorJmxReporterTest {
+
+  private MirrorJmxReporter mirrorJmxReporter;
+  private Metrics metrics;
+  private final String TEST_TOPIC = "TestTopic";
+
+  @Before
+  public void setUp() throws Exception {
+    mirrorJmxReporter = MirrorJmxReporter.getInstance();
+    metrics = mirrorJmxReporter.metrics;
+  }
+
+  @Test
+  public void updateLatencyMetrics() {
+    TopicPartition topicPartition = new TopicPartition(TEST_TOPIC, 1);
+    mirrorJmxReporter.addTopics(List.of(topicPartition));
+
+    mirrorJmxReporter.recordMirrorLatency(TEST_TOPIC, 500);
```

Comment:
Have you considered adding a comment here, to declare in which buckets these entries are expected to land?